config/routes.rb

```diff
@@ -28,6 +28,12 @@
     resources  :categories, only: :index
     resources  :prospective_users, only: :create
     resources  :assessments, only: [:index, :show, :update, :create] do
+      collection do
+        resources :shared, controller: 'shared_assessments', only: [:show], param: :token do
```

Comment:
This seems a bit strange to me - we're only using one resource on `shared_assessments` and using it as a namespace for other resources below.

We're not using the routes generated from this anywhere that I can see, but I'd like to see how open you would be to changing this up a bit?

Something like:

```
collection do
  get :show, to: 'shared_assessments#show', as: 'shared_assessments', path: '/shared/:token'
  get :report, to: 'shared_report#show', as: 'shared_reports', path: '/shared/:shared_token/report'
  get :priorities, to: 'shared_priorities#index', as: 'shared_priorities', path: '/shared/:shared_token/priorities'
end
```

This reads a bit more verbose, but the idea is that it provides the same routes.

Thoughts?